packages/fluxible/utils/callAction.js

```diff
@@ -0,0 +1,40 @@
+/**
+ * Copyright 2014, Yahoo! Inc.
+ * Copyrights licensed under the New BSD License. See the accompanying LICENSE file for terms.
+ */
+/* global Promise */
+'use strict';
+var isPromise = require('is-promise');
+
+/**
+ * Call an action supporting Promise expectations on invocation.
+ *
+ * If done callback supplied, that indicates non-Promise invocation expectation,
+ * otherwise, Promise invocation.
+ */
+function callAction (action, context, payload, done) {
+  if (done) {
```

Comment:
I think we don't want to short circuit here. It should always return a `Promise` just like `executeAction` does: https://github.com/yahoo/fluxible/blob/master/packages/fluxible/lib/FluxibleContext.js#L174

Did you see an issue that caused you to add this check?